src/components/Game.js

```diff
@@ -24,16 +24,20 @@ const Game = (props) => {
   // The global flow of tempo to facilitate the useEffect update loop
   const [canvasTimer, setCanvasTimer] = useState(0);
   const [gameController, setGameController] = useState(new GameController({}));
-  const [audioController, setAudioController] = useState(
-    new AudioController({})
-  );
-  const [settingsController, setSettingsController] = useState(
+    const [settingsController, setSettingsController] = useState(
     new SettingsController({})
   );
+  const [audioController, setAudioController] = useState(
+    new AudioController({settingsController: settingsController})
+  );
 
   const [gameState, setGameState] = useState(
     new GameState({
-      coreState: new CoreState({}),
+      coreState: new CoreState({
```

Comment:
Don't pass a CoreState into GameState; have GameState itself make it and pass its controllers to the core after that